packages/story-editor/src/app/media/types.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import type { FunctionComponent } from 'react';
+import type {
+  Resource,
+  ResourceId,
+  TrimData,
+  VideoResource,
+} from '@googleforcreators/media';
+import type { ElementId } from '@googleforcreators/elements';
+
+export const INITIAL_STATE = 'INITIAL_STATE';
+
+export enum ContentType {
+  Image = 'image',
+  Video = 'video',
+  Gif = 'gif',
+  Sticker = 'sticker',
+}
+
+export enum ProviderType {
```

Comment:
Maybe instead just convert [these constants](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/app/media/media3p/constants.js) into enums and use those directly since we'll need to do it anyway. Thoughts?

Reply:
That's the plan, yes, but haven't converted `getResourceFromMedia3p` or `packages/story-editor/src/app/media/media3p/providerConfiguration.js` yet, because then I also need to convert `packages/story-editor/src/app/media/media3p/attribution.js` which requires converting the `Text` component from design system for instance.

Also had issues with typing SVG imports as well in this PR, so this is a bit half-baked right now.

Reply:
Alright, I was thinking of just converting just that one `constants` file with the two enums.